src/System.IO.FileSystem/tests/FileInfo/GetSetAttributes.cs

```diff
@@ -122,5 +122,40 @@ public void WindowsInvalidAttributes(FileAttributes attr)
             Set(path, attr);
             Assert.Equal(FileAttributes.Normal, Get(path));
         }
+
+        // In NetFX we ignore "not found" errors, which leaves the attributes
+        // state as invalid (0xFFFFFFFF), which makes all flags true.
+
+        [Theory, MemberData(nameof(TrailingCharacters))]
+        public void GetAttributes_MissingFile(char trailingChar)
+        {
+            string path = GetTestFilePath();
+            FileInfo info = new FileInfo(path + trailingChar);
+            Assert.Equal((FileAttributes)(-1), info.Attributes);
```

Comment:
nit: same comment about using the Get/Set functions instead of directly invoking the Attributes property.